### PR TITLE
Uploading business emails

### PR DESCRIPTION
**Pull request by aarturoc**
New folders with the documentation.
19 -> this folder has info of Create functionality
20 -> this folder has info of Delete functionality
in this case a split de function in other to reduce responsibilities

principal README has the new links of the folder 19, 20

### CODE REVIEW

File: 19/README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+This email needs to be host in a place in order to use single sing on.
+
+### Information:
+- User corporate email
+
+>This information can be a list of multiple users
+
+### Create:
+- Insert email
+- Insert list of email
```
Comment (andymotagl) [inline 19/README.md:11]: I consider this must be on a separated document due the responsibility of create user is a single record, no multiple

File: 20/README.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+This email needs to be host in a place in order to use single sing on.
+
+### Information:
+- User corporate email
+
+>This information can be a list of multiple users
+
+### Delete:
+- Delete email per user
+- Delete email per business
```
Comment (andymotagl) [inline 20/README.md:11]: Let's drop this use case. only dropping a specific user will be supported in the meanwhile

File: README.md [hunk 3]
```diff
@@ -35,4 +35,6 @@ The documentation under this repository is oriented to offer a guide with genera
 | 15 | Filter global scores                                      |[15](/15/README.md)|
 | 16 | Draw global scores                                        |[16](/16/README.md)|
 | 17 | Get the experience attempts remaining                     |[17](/17/README.md)|
-| 18 | Take an experience                                        |[18](/18/README.md)|
\ No newline at end of file
+| 18 | Take an experience                                        |[18](/18/README.md)|
+| 19 | Create function for uploading business emails             |[19](/19/README.md)|
+| 20 | Delete function for Uploading business emails             |[20](/20/README.md)|
```
Comment (andymotagl) [inline README.md:40]: Use cases should not contain technical expressions such upload and function

**Review by andymotagl — COMMENTED** [review]
It looks good! 
I guess due to the misunderstanding about what an user is this got related that an email is different than an user but on a business POV is the same.

**Comment by aarturoc** [comment]
Ok, i have been working in these suggestions so I will close de PR

**Pull request closed without merging**